kubevirt-ami-centos.json

```diff
@@ -16,7 +17,7 @@
     "source_ami": "{{user `source_ami`}}",
     "instance_type": "{{user `instance_type`}}",
     "ssh_username": "centos",
-    "ami_name": "centos-kubevirt-{{timestamp}}",
+    "ami_name": "{{user `aws_instance_name`}}",
```

Comment:
Is the instance name kubevirt-centos-PR-X or kubevirt-centos-build-X?

Reply:
I've made it kubevirt-centos-PR-X if a pull request triggers the build. The build-X was what I named it when first testing.